kura/org.eclipse.kura.linux.watchdog/src/main/java/org/eclipse/kura/linux/watchdog/WatchdogServiceImpl.java

```diff
@@ -31,310 +31,310 @@
 
 public class WatchdogServiceImpl implements WatchdogService, ConfigurableComponent {
 
-    private static final Logger s_logger = LoggerFactory.getLogger(WatchdogServiceImpl.class);
```

Comment:
I am very sorry but I never had any details on the new code style formatter and I have no idea of what I should do...